File: minecraft/origin/src/main/java/com/github/theword/queqiao/event/minecraft/MinecraftPlayerJoinEvent.java
package com.github.theword.queqiao.event.minecraft;

import com.github.theword.queqiao.tool.event.base.BasePlayerJoinEvent;

public class MinecraftPlayerJoinEvent extends BasePlayerJoinEvent {
    public MinecraftPlayerJoinEvent(MinecraftPlayer player) {
        super("MinecraftPlayerJoinEvent", player);
    }
}

File: minecraft/origin/src/main/kotlin/com/github/theword/queqiao/event/minecraft/MinecraftPlayerJoinEvent.kt
package com.github.theword.queqiao.event.minecraft

import com.github.theword.queqiao.tool.event.base.BasePlayerJoinEvent

data class MinecraftPlayerJoinEvent(
    private val player: MinecraftPlayer
) : BasePlayerJoinEvent("MinecraftPlayerJoinEvent", player)